Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -151,6 +151,19 @@ public Action<InputUpdateType> onBeforeUpdate
             }
```

Comment:
Im getting a compile error on this branch, NativeInputRuntime.cs(16,41): error CS0535: 'NativeInputRuntime' does not implement interface member 'IInputRuntime.runInBackground'

Reply:
Do we have to land any native first?

Reply:
Good catch. Seems I committed incomplete changes (guess that is me being too used to hg workflows where last-minute changes to "staged" files will also be staged, whereas in git changes are "staged" on a per-change level, not per-file).

Fixed.